subsequent build command fails

The work flow:
1) Clone tensorflow_cc master (or v1.15.0 - same result right now);
2) Configure with CMake (tested on Ubuntu 18.04 with Make Generator, CUDA support);
3) Build (by either `cmake --build ...` or `make` directly;

The project being built successfully.

4) Build for the second time (call `make`):
The build is failing on patch step:
```
[ 56%] Performing configure step for 'tensorflow_shared'
v1.15.0-Rename-gettid-functions.patch:25: trailing whitespace.
 
v1.15.0-Rename-gettid-functions.patch:28: trailing whitespace.
 
v1.15.0-Rename-gettid-functions.patch:37: trailing whitespace.
 
v1.15.0-Rename-gettid-functions.patch:47: trailing whitespace.
 
v1.15.0-Rename-gettid-functions.patch:50: trailing whitespace.
 
error: third_party/Rename-gettid-functions.patch: already exists in working directory
CMakeFiles/tensorflow_shared.dir/build.make:107: recipe for target 'tensorflow-stamp/tensorflow_shared-configure' failed
```

---
A workaround:
```sh
cd tensorflow_cc/tensorflow
git reset HEAD
git checkout .
rm third_party/Rename-gettid-functions.patch v1.15.0-Rename-gettid-functions.patch v1.15.0-workspace.bzl.patch
```

Run build again: it succeed.

---
Would be nice to be able to run build again without manual steps.

--- a/tensorflow_cc/cmake/TensorflowShared.cmake
+++ b/tensorflow_cc/cmake/TensorflowShared.cmake
@@ -16,17 +16,15 @@ ExternalProject_Add(
         COMMAND cp "${CMAKE_CURRENT_SOURCE_DIR}/patches/v1.15.0-workspace.bzl.patch" .
         COMMAND git apply v1.15.0-workspace.bzl.patch
   CONFIGURE_COMMAND tensorflow/contrib/makefile/compile_linux_protobuf.sh
-            # Do not fail on warnings when building nsync
-            COMMAND sed -i "s/ -Werror//g" tensorflow/contrib/makefile/compile_nsync.sh
-            COMMAND tensorflow/contrib/makefile/compile_nsync.sh
-            COMMAND "${CMAKE_CURRENT_BINARY_DIR}/build_tensorflow.sh"
-            COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/cmake/copy_links.sh" .
-            # For some reason, Bazel sometimes puts the headers into
-            # `bazel-genfiles/genfiles` and sometimes just to `bazel-genfiles`.
-            # So we just create and include both the directories.
-            COMMAND mkdir -p bazel-genfiles/genfiles
-            COMMAND touch bazel-genfiles/_placeholder.h
-            COMMAND touch bazel-genfiles/genfiles/_placeholder.h
-  BUILD_COMMAND ""
-  INSTALL_COMMAND ""
+           # Do not fail on warnings when building nsync
+           COMMAND sed -i "s/ -Werror//g" tensorflow/contrib/makefile/compile_nsync.sh
+           COMMAND tensorflow/contrib/makefile/compile_nsync.sh
+  BUILD_COMMAND "${CMAKE_CURRENT_BINARY_DIR}/build_tensorflow.sh"
+  INSTALL_COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/cmake/copy_links.sh" .
+         # For some reason, Bazel sometimes puts the headers into
+         # `bazel-genfiles/genfiles` and sometimes just to `bazel-genfiles`.
+         # So we just create and include both the directories.
+         COMMAND mkdir -p bazel-genfiles/genfiles
+         COMMAND touch bazel-genfiles/_placeholder.h
+         COMMAND touch bazel-genfiles/genfiles/_placeholder.h
 )

--- a/tensorflow_cc/cmake/TensorflowShared.cmake
+++ b/tensorflow_cc/cmake/TensorflowShared.cmake
@@ -10,12 +10,12 @@ ExternalProject_Add(
   BUILD_IN_SOURCE 1
   DOWNLOAD_COMMAND ""
   UPDATE_COMMAND ""
-            # The grpc integrated in v1.15.0 needs patching, see https://github.com/clearlinux/distribution/issues/1151.
-  CONFIGURE_COMMAND cp "${CMAKE_CURRENT_SOURCE_DIR}/patches/v1.15.0-Rename-gettid-functions.patch" .
-            COMMAND git apply v1.15.0-Rename-gettid-functions.patch
-            COMMAND cp "${CMAKE_CURRENT_SOURCE_DIR}/patches/v1.15.0-workspace.bzl.patch" .
-            COMMAND git apply v1.15.0-workspace.bzl.patch
-            COMMAND tensorflow/contrib/makefile/compile_linux_protobuf.sh
+  # The grpc integrated in v1.15.0 needs patching, see https://github.com/clearlinux/distribution/issues/1151.
+  PATCH_COMMAND cp "${CMAKE_CURRENT_SOURCE_DIR}/patches/v1.15.0-Rename-gettid-functions.patch" .
+        COMMAND git apply v1.15.0-Rename-gettid-functions.patch
+        COMMAND cp "${CMAKE_CURRENT_SOURCE_DIR}/patches/v1.15.0-workspace.bzl.patch" .
+        COMMAND git apply v1.15.0-workspace.bzl.patch
+  CONFIGURE_COMMAND tensorflow/contrib/makefile/compile_linux_protobuf.sh
             # Do not fail on warnings when building nsync
             COMMAND sed -i "s/ -Werror//g" tensorflow/contrib/makefile/compile_nsync.sh
             COMMAND tensorflow/contrib/makefile/compile_nsync.sh

